Sidney-/slugtrade
# removed unnecessary files from repo

Changes:
- removed .idea directory from root directory of repo
- modified .gitignore to ignore .idea directory
- removed duplicate line .DS_Store from .gitignore

How to test:
- makes sure the server starts up
- make sure git doesn't detect any modified files after the server has been run
